src/rules/__tests__/unbound-method.test.ts

```diff
@@ -225,11 +248,14 @@ function addContainsMethodsClassInvalid(
 }
 
 ruleTester.run('unbound-method', requireRule(false), {
-  valid: [
+  valid: withFixtureFilename([
     'Promise.resolve().then(console.log);',
     "['1', '2', '3'].map(Number.parseInt);",
     '[5.2, 7.1, 3.6].map(Math.floor);',
     'const x = console.log;',
+    ...(parseInt(rawTypeScriptESLintPluginVersion.split('.')[0], 10) >= 6
```

Comment:
won't this always be true?

Reply:
(also, we should use `semver` module for this)

Reply:
No because we still support and test against `@typescript-eslint/eslint-plugin` v5

Reply:
I'll not replace this with `semver` since it'll just go away right after this PR when we drop support for v5